### PR TITLE
Adding $$LogSagas to global in react-native

### PR DESCRIPTION
**Pull request by benhughes**
Quick little check so the $$LogSagas can be added while debugging react-native

### CODE REVIEW

File: examples/sagaMonitor/index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const ERROR_STYLE = 'color: red'
 const CANCEL_STYLE = 'color: #ccc'
 
 const IS_BROWSER = (typeof window !== 'undefined' && window.document)
+const IS_REACT_NATIVE typeof window.document === 'undefined' && navigator.product === 'ReactNative'
```
Comment (Andarist) [inline examples/sagaMonitor/index.js]: you are missing an assignment here

File: examples/sagaMonitor/index.js [hunk 2]
```diff
@@ -392,6 +393,10 @@ if(IS_BROWSER) {
   window.$$LogSagas = logSaga
 }
 
+if (IS_REACT_NATIVE) {
```
Comment (Andarist) [inline examples/sagaMonitor/index.js]: could you refactor it, so both `IS_BROWSER` and `IS_REACT_NATIVE ` cases are handled at the same time? some kind of obtaining global object at the top and checking here if it got obtained

**Comment by benhughes** [comment]
How about this?

File: examples/sagaMonitor/index.js [hunk 3]
```diff
@@ -13,8 +13,13 @@ const EFFECT_TYPE_STYLE = 'color: blue'
 const ERROR_STYLE = 'color: red'
 const CANCEL_STYLE = 'color: #ccc'
 
+let globalScope = null;
+if (typeof global !== 'undefined')
```
Comment (Andarist) [inline examples/sagaMonitor/index.js]: could be made `const` with ternary operator like so:
```
const globalScope = (
  typeof window.document === 'undefined' && navigator.product === 'ReactNative' ? global
  : (typeof window !== 'undefined' && window.document) ? window
  : null
)
```

**Comment by benhughes** [comment]
Done, not a massive fan of nesting ternary expressions,, but seems to work OK.

**Comment by Andarist** [comment]
@benhughes 

Cool, could be merged, but would prefer if you squash those 4 commits into one, to keep the history cleaner. Additionally you can rebase too. Thanks in advance!

**Comment by benhughes** [comment]
Done, if any issues are raised from this please feel free to assign the issues to me. Thanks :)

**Comment by Andarist** [comment]
Thanks!

**Pull request merged**